apps/src/code-studio/pd/workshop_dashboard/workshops/workshop.module.scss

```diff
@@ -48,3 +48,135 @@
     }
   }
 }
+
+.card {
+  border: 1px solid var(--borders-neutral-primary);
+
+  .cardHeader {
+    border-bottom: 1px solid var(--borders-neutral-primary);
+    background-color: var(--background-neutral-secondary);
+    padding: 0.5rem 0.875rem;
+
+    .cardHeaderContainer {
+      display: flex;
+      align-items: center;
+      gap: 0.5rem;
+    }
+
+    .editButton {
+      margin-left: 0.5rem;
+    }
+  }
+
+  .cardContent {
+    padding: 0.75rem 1rem;
+
+    &:global(.MuiCardContent-root:last-child) {
+      padding-bottom: 1rem;
+    }
+  }
+
+  .divider {
+    color: var(--borders-neutral-primary);
+  }
+
+  .sectionContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    @media (min-width: 768px) {
+      flex-direction: row;
+    }
+  }
+
+  .column {
+    flex: 1;
+    min-width: 0;
+    gap: 0.875rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+  }
+
+  .unstyledList {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  .labelRow {
+    display: flex;
+    gap: 0.5rem;
+    margin-bottom: 0.25rem;
+    align-items: center;
+    max-height: 1.125rem;
+  }
+
+  .sessionRow {
+    display: flex;
+    gap: 0.5rem;
+
+    :nth-child(1) {
+      flex: 0 0 25%;
+    }
+
+    :nth-child(2) {
+      flex: 0 0 35%;
+    }
+
+    :nth-child(3) {
+      flex: 0 0 40%;
+    }
+  }
+
+  .workshopTag {
+    > div {
+      max-height: 1.125rem;
+      background-color: var(--background-neutral-tertiary);
+
+      span {
+        margin: 0;
+      }
+    }
+
+    &.visibility {
+      > div {
+        background-color: var(--background-neutral-tertiary);
+      }
+    }
+    &.status {
+      > div {
+        background-color: var(--background-brand-teal-primary);
+
+        span {
+          color: var(--text-neutral-white-fixed);
+        }
+      }
+    }
+  }
+
+  .workshopLink {
+    margin-top: 0.375rem;
+  }
+
+  :global(.MuiTable-root) {
+    width: unset;
+    margin-top: 0.5rem;
+
+    :global(.MuiTableCell-root) {
+      background-color: var(--background-neutral-primary);
+      border: none;
+      padding: 0.5rem 0;
+      padding-right: 1.5rem;
+    }
+  }
+}
+
+:global(.MuiDialogTitle-root).dialogTitle {
+  color: var(--text-neutral-primary);
+}
+
+:global(.MuiPaper-root).card {
+  box-shadow: 0 2px 3px 0 rgba(0 0 0 / 5%);
+}
```

Comment:
Can we avoid using general lavel selectors here and at least use selectors scoped to some specific module className (like for MuiTable-root)?

Also, why won't simple .dialogTitle and .card selectors won't work? Mui styles have as low priority as possible, meaning it should always be easy to overwrite them